django/forms/widgets.py

```diff
@@ -428,6 +431,9 @@ def value_from_datadict(self, data, files, name):
             return False
         return upload
 
+    def use_required_attribute(self, initial):
```

Comment:
It seems a bit awkward to pass `initial` to this method when it's only need for one widget. I'm not sure if there's a better approach. I thought of passing the entire `BoundField` which might be useful for other use cases, but that might end up with a poor separation of concerns. To be decided.